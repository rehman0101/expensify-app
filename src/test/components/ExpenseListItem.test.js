import React from 'react';
import ExpenseListItem from '../../components/ExpenseListItem';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';

test('should render expense list item with given data',()=>{
    const {id, description, amount, createdAt} = expenses[1];
    const wrapper = shallow(<ExpenseListItem
        id={id}
        description = {description}
        amount = {amount}
        createdAt = {createdAt}
     />);
     expect(wrapper).toMatchSnapshot();
})