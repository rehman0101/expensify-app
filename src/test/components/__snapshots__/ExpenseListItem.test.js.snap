// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list item with given data 1`] = `
<Link
  className="list-item"
  replace={false}
  to="/edit/2"
>
  <div>
    <h3
      className="list-item__title"
    >
      Rent
    </h3>
    <span
      className="list-item__subtitle"
    >
      December 28th, 1969
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    $195.00
  </h3>
</Link>
`;
